config/initializers/filter_parameter_logging.rb

```diff
@@ -2,6 +2,32 @@
 
 # Configure sensitive parameters which will be filtered from the log file.
 Rails.application.config.filter_parameters += [
-  :password, :password_confirmation,
-  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn,
+  :password,
```

Comment:
I did a quick query and here are maybe some others (there's a little overlap). Maybe garbage, but at least scan it:

```
Address1
Address2
City
DOB
EntryFromStreetESSH
FirstName
LastName
LastPermanentCity
LastPermanentState
LastPermanentStreet
LastPermanentZIP
Medicaid
Medicare
MiddleName
PreviousStreetESSH
UserEmail
UserFirstName
UserLastName
UserPhone
ZIP
Zip
address
address2
assessor_email
assessor_name
cell
cell_name
cell_phone
city
client
client_details
client_dob
client_email
client_emails
client_phones
client_race
contact_name
counselor_name
email
encrypted_FirstName
encrypted_FirstName_iv
encrypted_LastName
encrypted_LastName_iv
encrypted_MiddleName
encrypted_MiddleName_iv
encrypted_NameSuffix
encrypted_NameSuffix_iv
encrypted_SSN
encrypted_SSN_iv
encrypted_api_key
encrypted_api_key_iv
encrypted_content
encrypted_content_iv
encrypted_ftp_pass
encrypted_ftp_pass_iv
encrypted_import_errors
encrypted_import_errors_iv
encrypted_pass
encrypted_pass_iv
encrypted_password
encrypted_password_iv
encrypted_s3_access_key_id
encrypted_s3_access_key_id_iv
encrypted_s3_secret
encrypted_s3_secret_access_key
encrypted_s3_secret_access_key_iv
encrypted_s3_secret_iv
encrypted_zip_file_password
encrypted_zip_file_password_iv
encrypted_zip_password
encrypted_zip_password_iv
hiv_aids
hiv_aids_entry
hiv_aids_exit
hiv_aids_id
hiv_aids_indefinite_impairing
hiv_aids_latest
hiv_answer
hiv_positive
last_name
last_zipcode
middle
middle_initial
middle_name
name
nickname
phone
phone_alternate
phone_number
postal_code
preferred_name
recipient
report_name
requestor_email
requestor_name
requestor_phone
short_name
soundex_first
soundex_last
ssm
stable_housing_zipcode
staff_email
staff_name
work_phone
zip
zip_code
zip_codes
zip_override
```

Reply:
Thanks!  I _think_ the match is on partial, so maybe we can streamline this a bunch (zip, instead of all three versions, etc.)  Do you know if that's the case?

Reply:
I don't know, but I'd throw a dart and expect it match fully, but case-insensitively.